basic/exercise11/scriptSpec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+
+describe("Checking if  degrees Celsius ", function(){
+    it("is degrees Fahrenheit", function(){
+        var temperatureInDegreesFahrenheit =  convertTemperaturesFromCelsiusToFahrenheit(degreesCelsius);
+        expect(temperatureInDegreesFahrenheit).toBeTruthy();
+    });
+});
+
+describe("Checking if  degrees Fahrenheit ", function(){
+    it("is degrees Celsius", function(){
+        var temperatureInDegreesCelsius =  convertTemperaturesFromFahrenheitToCelsius(degreesFahrenheit);
+        expect(temperatureInDegreesCelsius).toBeTruthy();
```

Comment:
If you check if it is truthy it means that you check if function returned true or false.
Is this what you function should return?
You should specify assertion that would compare function result to concrete value.